browser/resources/settings/brave_new_tab_page/brave_new_tab_page.html

```diff
@@ -0,0 +1,71 @@
+<link rel="import" href="chrome://resources/html/polymer.html">
+
+<link rel="import" href="chrome://resources/html/i18n_behavior.html">
+<link rel="import" href="chrome://resources/cr_elements/md_select_css.html">
+<link rel="import" href="brave_new_tab_browser_proxy.html">
+<link rel="import" href="../settings_shared_css.html">
+<link rel="import" href="../settings_vars_css.html">
+
+<dom-module id="settings-brave-new-tab-page">
+  <template>
+    <style include="settings-shared md-select iron-flex">
+      .primary-title {
+        font-weight: 700;
+        padding-top: 1em;
+        font-size: 1.05em;
+      }
+      .nested-section {
+        margin-inline-start: 1.5em;
+        margin-top: -0.5em;
+      }
+      .borderless {
+        border: none;
+      }
+      .normalize-margin {
+        margin: 0 -1.5em 1em;
+      }
+    </style>
+    <div class="settings-box first">
+      <div class="flex">
+        <div class="label primary-title">$i18n{braveNewTabCustomizeDashboard}</div>
+        <div class="normalize-margin">
+          <settings-toggle-button
+              class="secondary borderless"
+              pref="{{prefs.brave.new_tab_page.show_background_image}}"
+              label="$i18n{braveNewTabBackgroundImages}">
+          </settings-toggle-button>
+          <!-- TODO: this toggle should be disabled if
+                prefs.brave.new_tab_page.show_branded_background_image is false -->
+          <settings-toggle-button
```

Comment:
I think this button should be disabled when `show_background_image` is false?

Reply:
Yes, correct. I had written a comment here asking for your advice but forgot to add. 

As you said and [for parity with this line](https://github.com/brave/brave-core/blob/master/components/brave_new_tab_ui/containers/newTab/settings.tsx#L124) the branded wallpaper should be disabled. I couldn't find a similar example here and still reading through Chromium docs. Do you know how can I conditionally disable this button? It sems it doesn't parse strings the same way as `pref` does

Reply:
Wow! great idea 👍

Reply:
IMO, I think just disabling sponsored images button would be better when bg option goes off because user only changed bg setting not the SI option. WDYT?